package.json

```diff
@@ -13,29 +13,29 @@
   },
   "main": "index",
   "engines": {
-    "node": "12 || 14"
```

Comment:
Using node versions higher than 18.15.0 will cause installation of certain plugins to fail, due to https://github.com/ZJONSSON/node-unzipper/issues/271. (after uploading and unzipping, properties.schema will be mangled, causing JSON.parse to fail).

Reply:
Have added your comment as an issue (https://github.com/adaptlearning/adapt_authoring/issues/2735) for discussion with the dev team :)